app/assets/javascripts/code/services/google/_map.js

```diff
@@ -185,10 +185,8 @@ export const map = (
     },
 
     removeAllMarkers: function() {
-      var markers = this.markers;
-      _(markers).each(function(marker) {
-        marker.setMap(null);
-      });
+      this.markers.forEach(marker => marker.setMap(null));
```

Comment:
I guess `marker.setMap(null)` would still work?

Reply:
Is there a chance `this.markers` is `null` or something like that? Cause `_()` handles that case and now would explode.